SConstruct

```diff
@@ -119,6 +125,30 @@ def update_rpm_version(version, tag):
 
     return True
 
+def update_debian_version(version, tag):
+    """ Update the version (and release) in the Debian changelog """
+    # pylint: disable=import-outside-toplevel
+    import email.utils
+    import debian.changelog
```

Comment:
(pylint-no-name-in-module) No name 'changelog' in module 'debian'
(pylint-import-error) Unable to import 'debian.changelog'